### PR TITLE
Fixed ‘CreateNewAnimation’ window wrong size.

### PR DESCRIPTION
**Pull request by ageazrael**

### CODE REVIEW

File: editor/plugins/animation_player_editor_plugin.cpp [hunk 1]
```diff
@@ -1377,19 +1377,21 @@ AnimationPlayerEditor::AnimationPlayerEditor(EditorNode *p_editor) {
...
 	l->set_text(TTR("Animation Name:"));
 	l->set_position(Point2(10, 10));
-
-	name_dialog->add_child(l);
+	vb->add_child(l);
 	name_title = l;
 
+	name = memnew(LineEdit);
+	vb->add_child(name);
+	name->set_position(Point2(18, 30));
```
Comment (bojidar-bg) [inline editor/plugins/animation_player_editor_plugin.cpp]: I think you shouldn't be doing this -- the container is supposed to manage the positions of its children.

Comment (ageazrael) [inline editor/plugins/animation_player_editor_plugin.cpp]: You right. I'll check it。

**Comment by akien-mga** [comment]
Could you rebase to squash the commits together? docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase

**Comment by akien-mga** [comment]
Merged manually as a89658f.

**Pull request closed without merging**